ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5692,3 +5693,1101 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: tags
+    uris: "{{ discussions_prefix }}/tags"
+    upstream_url: "{{ discussions_mw_service_url }}/tags"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```

Comment:
does the API create tags or will list or read them?

Reply:
Its the read call.

Reply:
then please update the ACL as contentAccess